src/snippet.rs

```diff
@@ -108,7 +108,7 @@ impl<'a> Snippet<'a> {
 /// An annotation for a [`Snippet`].
 ///
 /// See [`Level::span`] to create a [`Annotation`]
-#[derive(Debug)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
Should likely split these API level changes out into their own commit with the motivations

Reply:
I was able to remove them entirely